packages/rspack/tests/Compiler.test.ts

```diff
@@ -335,6 +335,11 @@ describe("Compiler", () => {
 		} catch (err) {
 			expect(err.toString()).toMatchInlineSnapshot(`
 			"Error:   × Resolve error: Can't resolve './missing-file' in '<PROJECT_ROOT>/tests'
+			  help: try to resolve './missing-file' in '<PROJECT_ROOT>/tests'
```

Comment:
This is too verbose, the default error message should be easy to read and the fix should be easy to apply.

Reply:
I have gave the useful messages. But I am not sure whether it is good enough. Help me to review. Thanks.